pandas/_libs/interval.pyi

```diff
@@ -9,7 +9,7 @@ import numpy as np
 import numpy.typing as npt
 
 from pandas._typing import (
-    IntervalClosedType,
+    IntervalInclusiveType,
```

Comment:
Hey, sorry the explanation during the sprint might have been confusing. I was referring to adding ``IntervalClosedType`` where this is not added yet, not renaming. Sorry for this.